service/src/main/kotlin/fi/espoo/evaka/invoicing/data/FeeDecisionQueries.kt

```diff
@@ -861,3 +861,32 @@ FROM partner_children, partner_fridge_children
             .bind("childIds", childIds)
             .bind("dateRange", dateRange)
             .exactlyOne<Boolean>()
+
+fun Database.Read.getFeeDecisionByLiableCitizen(
+    citizenId: PersonId
+): List<FeeDecisionCitizenInfoRow> {
+    return createQuery(
+            """
+SELECT fd.id,
+       fd.valid_during,
+       fd.sent_at,
+       fd.head_of_family_id,
+       fd.partner_id
+FROM fee_decision fd
+WHERE fd.status in ('SENT')
+AND (head_of_family_id = :citizenId
+    OR partner_id = :citizenId)
+    """
```

Comment:
Perhaps this query could check that `document_key IS NOT NULL`, because downloading the PDF will fail otherwise. Lots of decisions are generated in the beginning of the month and there may be a delay, and the citizen will only get a message after the pdf has been generated anyway.

Reply:
Added filtering based on document key.